### ISSUE TITLE
a11y: Login view

### ISSUE THREAD

**Issue by woodruffw** [pypa/warehouse#6040]
Issues in `/account/login`:

* Explicit positive `tabindex` values [should not be used](https://www.scottohara.me/blog/2019/05/25/tabindex.html) in the login form's inputs. Omitting `tabindex` entirely should be fine in this case.

**Comment by anowlcalledjosh** [comment]
This means that the password field now takes two <kbd>Tab</kbd>-presses to reach from the username field rather than one, which breaks at least one popular password manager.

If the accessibility issue is more important than avoiding some manual configuration of password managers, then fine (I'm not qualified to make that decision), but the original change to add `tabindex`es was intentional – see #2824, #2917.

**Comment by di** [comment]
This has actually been bugging me too. I think we can fix it without adding tab indexes by moving the "Show password" box under the password field.

@nlhkabu @woodruffw Thoughts?

**Comment by woodruffw** [comment]
> I think we can fix it without adding tab indexes by moving the "Show password" box under the password field.

Yep, I think this would be a good solution.
